src/backend/legendary/library.ts

```diff
@@ -486,6 +486,8 @@ export class LegendaryLibrary {
     const CloudSaveFolder = customAttributes?.CloudSaveFolder
     const FolderName = customAttributes?.FolderName
     const canRunOffline = customAttributes?.CanRunOffline?.value === 'true'
+    const thirdPartyManagedApp =
```

Comment:
Thinking about this, maybe we should just store all `customAttributes` in the `GameInfo`

Anyways, for now this is fine

Reply:
yes, lets think about it in the future because it also will have downsides if the props wont be used anywhere.